message/tests.py

```diff
@@ -0,0 +1,82 @@
+from django.contrib.auth import get_user_model
+from player.models import Player, BallGame
+from game_event.models import GameEvent
+from message.models import Message
+from django.core.exceptions import ValidationError
+from datetime import datetime
+import pytest
+
+
+@pytest.mark.django_db
+class TestMessageModel:
+
+    @pytest.fixture
+    def user(self):
+        return get_user_model().objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+
+    @pytest.fixture
+    def saved_player(self, user):
+        player = Player.objects.create(
+            user=user,
+            birth_date='1990-01-01',
+            favorite_ball_game=BallGame.Soccer
+        )
+        return player
+
+    @pytest.fixture
+    def saved_game_event(self):
+        game_event = GameEvent.objects.create()
+        return game_event
+
+    @pytest.fixture
+    def saved_message(self, saved_player, saved_game_event):
+        message = Message.objects.create(
+            user_id=saved_player,
+            game_event_id=saved_game_event,
+            time_sent=datetime(2023, 4, 18, 12, 30, 0),
+            message_text="test message"
+        )
+        return message
+
+    def test_get_message(self, saved_message):
+        message = Message.objects.get(pk=saved_message.pk)
+        assert message == saved_message
+
+    def test_update_message(self, saved_message):
+        message = saved_message
+        assert message.message_text != "test message?!"
+        message.message_text = "test message?!"
+        message.save()
+        updated_message = Message.objects.get(pk=message.pk)
+        assert updated_message.message_text == "test message?!"
+
+    def test_delete_message(self, saved_message):
+        message = saved_message
+        message.delete()
+        with pytest.raises(Message.DoesNotExist):
+            Message.objects.get(pk=saved_message.pk)
+
+    def test_delete_game_event_deletes_message(self, saved_message):
```

Comment:
you can do the same test with delete user

Reply:
@Yuval-Vino We are already in confusion if when deleting a user we also need to delete his message. wdyt ?

Reply:
It's up to you. Currently, the model says it should be deleted (as they are cascaded on delete), so I'd say implement according to the model you have.

Reply:
@yftacherzog @Yuval-Vino we add the relevant test for this case in accordance with what our model says.